packages/backend/src/services/GitIntegrationService/GitIntegrationService.ts

```diff
@@ -466,4 +479,131 @@ Affected charts:
             projectUuid,
         });
     }
+
+    private static async createSqlFile({
+        githubProps,
+        name,
+        sql,
+    }: {
+        githubProps: GithubProps;
+        name: string;
+        sql: string;
+    }): Promise<any> {
+        const fileName = `models/${friendlyName(name)}.sql`;
+        const content = `
+{{
+  config(
+    tags=['lightdash-write-back']
+  )
+}}
+  
+${sql}
+`;
+
+        return createFile({
+            ...githubProps,
+            fileName,
+            content,
+            message: `Created file ${fileName} `,
+        });
+    }
+
+    private static async createYmlFile({
+        githubProps,
+        name,
+        columns,
+    }: {
+        githubProps: GithubProps;
+        name: string;
+        columns: VizColumn[];
+    }): Promise<any> {
```

Comment:
here we can remove this explicit type definition and apply the return type of `createFile` as referenced above ⬆️